src/mudlet-lua/lua/Other.lua

```diff
@@ -183,10 +183,7 @@ end
 function appendScript(name, luaCode, pos)
   pos = pos or 1
   assert(type(name) == "string", "appendScript: bad argument #1 type (script name as string expected, got "..type(name).."!)")
-  assert(type(name) == "string", "appendScript: bad argument #2 type (lua code as string expected, got "..type(luaCode).."!)")
-  if not getScript(name, pos) then
-    return nil, 'script "'..name..'" at position "'..pos..'" not found'
-  end
+  assert(type(luaCode) == "string", "appendScript: bad argument #2 type (lua code as string expected, got "..type(luaCode).."!)")
   return setScript(name, getScript(name, pos).."\n"..luaCode, pos)
```

Comment:
Hang on - you have changed `getString(...)` in `TLuaInterpreter.cpp` to now return a `-1` and an error message as a *run-time* error if the script is NOT found. That is fine, but you MUST handle it correctly here, I think in that case that `-1` gets *coerced* into a `"-1"` so that the new script is added to it so that you get a Lua script of the form:
```lua
-1
<what is in the luaCode string>
```
I don't think that is what you want! :stuck_out_tongue_closed_eyes:

Reply:
Yes, thats obviously not what I want, but that is never happening.
getScript() only would return -1 if it cannot find the script. 

So if getScript cannot find the script also setScript cannot find it and it will return
the error message _-1 "script "yourscript" at position "yourposition" not found"_.

Under normal circumstances it should be impossible that getScript doesn't find the script and setScript does. 
That would mean that I made something wrong somewhere else and then the whole append function wouldn't work anyway.

Maybe the name appendScript for the function  is confusing? (I'm aware that my English is not to good)
I could change it to a less confusing one if you have some suggestion.

Otherwise what would you suggest? Should I return to the previous code and check if getScript found the script before appending?